store/mockstore/mocktikv/cop_handler_dag.go

```diff
@@ -169,8 +124,9 @@ func (h *rpcHandler) buildDAGExecutor(req *coprocessor.Request) (*dagContext, ex
 // timezone offset in seconds east of UTC is used to constructed the timezone.
 func constructTimeZone(name string, offset int) (*time.Location, error) {
 	if name != "" {
-		return LocCache.getLoc(name)
+		return timeutil.LoadLocation(name)
 	}
+
 	return time.FixedZone("", offset), nil
```

Comment:
When will the code run to this branch and what happen then ?

Reply:
user runs `set time_zone="+8:00"`?

Reply:
But the time zone name is not "UTC" ?

Reply:
In this case, the name should be "empty".